### PR TITLE
update ui docker enntry point script.

### LINKED ISSUE #7401: 8.7.0 UI docker image error
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Bug

___
### Bug
- Which version of SkyWalking, OS, and JRE?
Skywalking:8.7.0
OS:Docker

- What happened?
After deployment through docker, a blank page will appear when accessing through the browser, and the errors reported on the console are as follows
```shell
2021-08-03 03:29:17.487 ERROR 1 --- [or-http-epoll-3] a.w.r.e.AbstractErrorWebExceptionHandler : [1816cafd-1]  500 Server Error for HTTP POST "/graphql"

io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:12800
        Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
        |_ checkpoint ⇢ org.springframework.cloud.gateway.filter.WeightCalculatorWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ org.springframework.boot.actuate.metrics.web.reactive.server.MetricsWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ HTTP POST "/graphql" [ExceptionHandlingWebHandler]
Stack trace:
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
        at io.netty.channel.unix.Errors.newConnectException0(Errors.java:155)
        at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:128)
        at io.netty.channel.unix.Socket.finishConnect(Socket.java:278)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:710)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:687)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567)
        at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:470)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)

2021-08-03 03:29:23.184 ERROR 1 --- [or-http-epoll-6] a.w.r.e.AbstractErrorWebExceptionHandler : [d4bc1a05-1]  500 Server Error for HTTP POST "/graphql"

io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:12800
        Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
        |_ checkpoint ⇢ org.springframework.cloud.gateway.filter.WeightCalculatorWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ org.springframework.boot.actuate.metrics.web.reactive.server.MetricsWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ HTTP POST "/graphql" [ExceptionHandlingWebHandler]
Stack trace:
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
        at io.netty.channel.unix.Errors.newConnectException0(Errors.java:155)
        at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:128)
        at io.netty.channel.unix.Socket.finishConnect(Socket.java:278)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:710)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:687)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567)
        at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:470)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)

2021-08-03 03:29:43.656 ERROR 1 --- [or-http-epoll-7] a.w.r.e.AbstractErrorWebExceptionHandler : [816cd19d-2]  500 Server Error for HTTP POST "/graphql"

io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:12800
        Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
        |_ checkpoint ⇢ org.springframework.cloud.gateway.filter.WeightCalculatorWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ org.springframework.boot.actuate.metrics.web.reactive.server.MetricsWebFilter [DefaultWebFilterChain]
        |_ checkpoint ⇢ HTTP POST "/graphql" [ExceptionHandlingWebHandler]
Stack trace:
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
        at io.netty.channel.unix.Errors.newConnectException0(Errors.java:155)
        at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:128)
        at io.netty.channel.unix.Socket.finishConnect(Socket.java:278)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:710)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:687)
        at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567)
        at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:470)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:378)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

The docker deployment script is as follows
```shell
docker pull apache/skywalking-oap-server:8.7.0-es7

docker run --name oap-server --restart always \
 -p 11800:11800 -p 12800:12800 \
 --net=sky-fly \
 -e SW_STORAGE=elasticsearch7 \
 -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 \
 -d apache/skywalking-oap-server:8.7.0-es7

docker pull apache/skywalking-ui:8.7.0

docker run --name oap-ui --restart always \
 -p 12889:8080 \
 --net=sky-fly \
 -e SW_OAP_ADDRESS=oap-server:12800 \
 -d apache/skywalking-ui:8.7.0
```

With the same deployment method, there is no problem with 8.6.0

___
### Requirement or improvement
Bug or my fault?

### PR DESCRIPTION
**Pull request by JaredTan95**
fix https://github.com/apache/skywalking/issues/7401

### CODE REVIEW

**Review by hanahmily — COMMENTED** [review]
`SW_OAP_ADDRESS` in `ui/Dockefile` should be fixed correspondingly.

File: ui/docker-entrypoint.sh [hunk 1]
```diff
@@ -20,7 +20,12 @@ set -e
...
-[[ ! -z "$SW_TIMEOUT" ]] && export COLLECTOR_RIBBON_READTIMEOUT=${SW_TIMEOUT} && echo "COLLECTOR_RIBBON_READTIMEOUT=$COLLECTOR_RIBBON_READTIMEOUT"
+if [[ ! -z "$SW_OAP_ADDRESS" ]]; then
+  address_arr=(${SW_OAP_ADDRESS//,/ })
+  for i in "${!address_arr[@]}"
+  do
+      JAVA_OPTS="${JAVA_OPTS} -Dspring.cloud.discovery.client.simple.instances.oap-service[$i].uri=${address_arr[$i]}"
+  done
+fi
 
-exec java -jar webapp/skywalking-webapp.jar "$@"
```
Comment (hanahmily) [inline ui/docker-entrypoint.sh:26]: `-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap` are invalid in jdk11

Comment (JaredTan95) [reply ui/docker-entrypoint.sh:26]: Okay, This dockerfile used jdk11, skywalking main repo used jdk8, I did not noticed that.

**Comment by hanahmily** [comment]
I found that docker-compose.yml doesn't pick up the skywalking version from the bash script, which causes we can't verify the images just built. Could you parameterize them?

File: ui/docker-entrypoint.sh (hunk 1 above)
Comment (hanahmily) [reply ui/docker-entrypoint.sh:26]: The main repo UI image is base on using jdk8, pls upgrade it to jdk11 if you want to remove them 😄

Comment (JaredTan95) [reply ui/docker-entrypoint.sh:26]: > The main repo UI image is base on using jdk8, pls upgrade it to jdk11 if you want to remove them 😄

I will deal with it next soon.

**Comment by JaredTan95** [comment]
> I found that docker-compose.yml doesn't pick up the skywalking version from the bash script, which causes we can't verify the images just built. Could you parameterize them?

Done

**Review by hanahmily — APPROVED** [review]
LGTM. Once this gets merged, pls re-push the 8.7.0 UI image.

**Comment by wu-sheng** [comment]
@JaredTan95 Do you have access to the docker hub repo?

**Comment by JaredTan95** [comment]
> @JaredTan95 Do you have access to the docker hub repo?

okay, I will deal with it after applied for permission.

**Pull request merged**